File: 프로그래머스/0/120847. 최댓값 만들기 （1）/최댓값 만들기 （1）.java
class Solution {
    public int solution(int[] numbers) {
        int answer = 0;
        int temp = 0;
        int length = numbers.length - 1;
        for(int i = 0; i < length; i++){ // 끝번호까지와 그 전번호까지만 가면되니까 idex가 for문에 미만을 하고
            // i가 커질수록 비교할 뒤의 인덱스들도 밀려나니까 커진만큼 빼줘야, 비교할 인덱스가 맞음
            for(int j = 0; j < length - i; j++){
                if(numbers[j] > numbers[j+1]){  
                    temp = numbers[j]; 
                    numbers[j] = numbers[j+1];
                    numbers[j+1] = temp;
                }
            }
        }
        answer = numbers[length] * numbers[length-1];
        return answer;
    }
}

File: 프로그래머스/0/120847. 최댓값 만들기 （1）/최댓값 만들기 （1）.py
def solution(numbers):
    length = len(numbers)
    for i in range(length - 1):
        for j in range(length - 1 - i):
            if numbers[j] > numbers[j + 1]:
                numbers[j], numbers[j + 1] = numbers[j + 1], numbers[j]

    answer = numbers[length - 1] * numbers[length - 2]
    return answer
